#: __init__.py:100
msgid "NextGIS geoinformation system"
msgstr "Geoinformační systém NextGIS"

#: __init__.py:225
msgid "Linux kernel"
msgstr "Linuxové jádro"

#: __init__.py:228
msgid "OS distribution"
msgstr "Distribuce OS"

#: __init__.py:240
msgid "CPU"
msgstr "CPU"

#: __init__.py:244
msgid "RAM"
msgstr "RAM"

#: exception.py:85
msgid "Validation error"
msgstr "Chyba ověření"

#: exception.py:90
msgid "Forbidden"
msgstr "Zakázán"

#: exception.py:95
msgid "Insufficient permissions"
msgstr "Nedostatečná oprávnění"

#: exception.py:100
msgid "Operational error"
msgstr "Provozní chyba "

#: exception.py:105
msgid "External service error"
msgstr "Chyba externí služby"

#: storage.py:258
msgid "Storage full"
msgstr "Skladování plné"

#: storage.py:259
#, python-format
msgid ""
"This operation can't be performed because the storage limit (%s) has been "
"already exceeded by %s."
msgstr ""
"Tuto operaci nelze provést, protože limit úložiště (%s) byl již překročen o "
"%s."

#: storage.py:267
msgid "Not enough storage"
msgstr "Nedostatek úložného prostoru"

#: storage.py:268
#, python-format
msgid ""
"This operation requires %s of storage and can't be performed because only %s "
"available."
msgstr ""
"Tato operace vyžaduje %s úložiště a nelze ji provést, protože je k dispozici "
"pouze %s."

#: storage.py:276
msgid "Estimation is already running"
msgstr "Odhad již běží"

#: test/test_exception.py:75
#, python-format
msgid "The answer is %d"
msgstr "Odpověď je %d"

